pkg/core/namespace.go

```diff
@@ -97,13 +99,17 @@ func (c *kubectlClient) NamespaceInit(manifests map[string]*Manifest, options Na
 		return err
 	}
 
+	ownerLabel := &kustomize.Label{
```

Comment:
might also be nice to include the version of riff as well

Reply:
done